public/app/plugins/panel/geomap/README.md

```diff
@@ -1,3 +1,3 @@
 # Geomap Panel - Native Plugin
 
-The Geomap is **included** with Grafana.
+The Geomap Panel is **included** with Grafana. Please see [Geomap Panel documentation](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/geomap/) for information on the latest features.
```

Comment:
I think for linking to internal documentation the `relref` option is preferred. E.g.
```
[documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}})
```